src/ngraph/runtime/cpu/op/dequantize.cpp

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+* Copyright 2018 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "ngraph/runtime/cpu/op/dequantize.hpp"
+#include "ngraph/op/constant.hpp"
+
+ngraph::op::Dequantize::Dequantize(std::shared_ptr<Node> input,
+                                   std::shared_ptr<Node> min,
+                                   std::shared_ptr<Node> max,
+                                   const element::Type& type)
+    : RequiresTensorViewArgs("Dequantize", {input, min, max})
+    , m_element_type(type)
+{
+    add_output(element::f32, input->get_shape());
```

Comment:
As with #1484, if we elevate this to core we'll need to check the element types and sizes of `min`/`max`, check the element type of `input` to make sure it is a valid type for a quantized, and make sure that `type` is supported.

Reply:
we at least should check that `min` and `max` are the same type and they are both scalars and maybe that the output type is f32/f64 and the inputs are integral types?

Reply:
Also check that min and max are constants.

Reply:
Shouldn't we be using type instead of element::f32?